scripts/g.extension/g.extension.py

```diff
@@ -396,7 +396,17 @@ def _get_addons_list(self):
                         )
         return addons_dict
 
+    def _addon_exists(self, addon_list):
+        if not [self.addons[addon] for addon in addon_list if addon in self.addons]:
+            gs.fatal(
+                _(
+                    "Add-On name <{}> doesn't exists in the repository."
+                    " Try it again with correct Add-On name, please."
```

Comment:
This part of the message is not needed and could be removed.
```suggestion
                    " Try it again with correct AddOn name, please."
```

Reply:
Yes, I agree.